packages/augur-sdk/src/api/Events.ts

```diff
@@ -12,7 +12,7 @@ export class Events {
     this.provider.storeAbiData(abi.Augur as Abi, "Augur");
   }
 
-  async getLogs(eventName: string, fromBlock: number, toBlock: number, additionalTopics?: Array<string | string[]>): Promise<ParsedLog[]> {
+  async getLogs(eventName: string, fromBlock: number, toBlock: number | "latest", additionalTopics?: Array<string | string[]>): Promise<ParsedLog[]> {
```

Comment:
not a fan of mixing types number and string seems odd. Is there any other approach available?